examples/realTimeVideo-config/src/main.ts

```diff
@@ -0,0 +1,94 @@
+import { Authentication, Fleet, KeyValue } from "@formant/data-sdk";
+import "@formant/ui-sdk-joystick";
+import "@formant/ui-sdk-realtime-player";
+import { RealtimePlayer } from "@formant/ui-sdk-realtime-player";
+import "./style.css";
+
+const loadVideo = async (defaultCamera: number) => {
+  (el("formant-realtime-player") as RealtimePlayer).drawer.start();
```

Comment:
`(el("formant-realtime-player") as RealtimePlayer)` gets used a few times through this function. Does it _need_ to be re-selected from the DOM each time, or can you hold onto a single DOM element for the scope of this method?

```ts
const player = el("formant-realtime-player") as RealtimePlayer;
```

and just reuse it later…

Reply:
in fact, it might be worth _genericifying_ `el()` a little bit to make this is a little more _enforced_:

```ts
const el = <T extends Element = HTMLElement>(selector: string) => {
  const match =  document.querySelector(selector);
  if (!match || !(match instanceof T)) {
    throw new Error(`"${selector}" not found/not correct type`;
  }
  return match;
}
```

Then you could just say here:

```ts
const player = el<RealtimePlayer>("formant-realtime-player");
```